addons/udes_sale_stock/tests/test_sale_order_state.py

```diff
@@ -0,0 +1,110 @@
+from . import common
+
+
+class TestSaleOrderState(common.BaseSaleUDES):
+    @classmethod
+    def setUpClass(cls):
+        super(TestSaleOrderState, cls).setUpClass()
+
+        Customer = cls.env["res.partner"]
+
+        cls.customer = Customer.create(
+            {
+                "name": "Bob",
+                "street": "1 Bobs House",
+                "street2": "Bobs street",
+                "city": "The city of Bob",
+                "zip": "EN1 4LS",
+            }
+        )
+
+        cls.quant = cls.create_quant(
+            cls.apple.id, cls.test_location_01.id, 30, package_id=cls.create_package().id
+        )
+
+        # Create sale order
+        sale = cls.create_sale(cls.customer)
+        cls.sale_line = cls.create_sale_line(sale, cls.apple, 15)
+        sale.requested_date = "2020-01-03"
```

Comment:
This can be added as a parameter at `create_sale`. Is this to avoid datetime conversions?

Reply:
It was a carry over from test_cancel_unfulfillable.py. I'll change it